addons/fix-editor-comments/addon.json

```diff
@@ -52,6 +52,18 @@
       "id": "standalone-edit",
       "type": "boolean",
       "default": true
+    },
+    {
+      "name": "Always spawn block comments next to block",
+      "id": "leash",
+      "type": "boolean",
+      "default": true
+    },
```

Comment:
Are we sure all old users of this addon will be fine with this being enabled automatically?

Reply:
Leashing does not happen that often, and it's easy to trace the behavior to this addon, so I'd say it's fine.

Reply:
Very few people would want a new comment to be created off-screen automatically. There is little purpose to it, and it is extremely annoying in most cases.